test/test_type_promotion.py

```diff
@@ -932,53 +932,6 @@ def test_integer_addcdiv_deprecated(self, device, dtype):
         with self.assertRaisesRegex(RuntimeError, '^Integer division.+is no longer supported+'):
             t.addcdiv_(t, t)
 
-    def _ternary_promotion_common(self, device, op1, op2):
```

Comment:
why is type promotion test removed?

Reply:
it's handled by the opinfo tests now. see this discussion with mario: https://github.com/pytorch/pytorch/pull/80842#discussion_r915740206